openfisca_france_dotations_locales/variables/agglomeration.py

```diff
@@ -45,7 +46,7 @@ class part_population_agglomeration_departement(Variable):
     def formula(commune, period, parameters):
         population_dgf_agglomeration = commune("population_dgf_agglomeration", period)
         population_dgf_departement_agglomeration = commune("population_dgf_departement_agglomeration", period)
-        return population_dgf_agglomeration / population_dgf_departement_agglomeration
+        return (population_dgf_departement_agglomeration > 0) * np.divide(population_dgf_agglomeration, population_dgf_departement_agglomeration, where = (population_dgf_departement_agglomeration > 0))
```

Comment:
Pourquoi indiquons-nous deux fois la condition ? En terme de calcul, np.divide optimise-t-il pas déjà en fonction de sa propriété `where` ?
```suggestion
        return np.divide(population_dgf_agglomeration, population_dgf_departement_agglomeration, where = (population_dgf_departement_agglomeration > 0))
```

Testé sous ipython et cela renvoie bien 0. lorsque le dénominateur est à zéro.